app/client/playwright.config.ts

```diff
@@ -0,0 +1,97 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
```

Comment:
Out of curiosity, why is this file ts?  Is that what the example did?  It's a bit odd to have a config file in typescript.

Reply:
I used their typescript initialization and that part was set up by default.

Reply:
Okay, makes sense.